xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-xwiki/src/main/java/org/xwiki/rendering/internal/wiki/XWikiWikiModel.java

```diff
@@ -312,7 +312,7 @@ private String getImageDimension(String dimension, Map<String, String> imagePara
                 value = sd.getPropertyValue(dimension);
             } catch (Exception e) {
                 // Ignore the style parameter but log a warning to let the user know.
-                this.logger.warn("Failed to parse CSS style [{}]", style);
+                this.logger.warn("Failed to parse CSS style [{}]", style, e);
```

Comment:
We tend to only log a `org.apache.commons.lang3.exception.ExceptionUtils.getRootCauseMessage(e)` in the case of a warning (a printed exception is pretty huge in the log, so we do that only for errors).

Reply:
Thanks @tmortagne for clarifying this coding rule.

Actually what I'm might do is deploying this custom version only in our development environment, reproduce the issue described in [XWIKI-20546](https://jira.xwiki.org/browse/XWIKI-20546) and add the log in the Jira ticket comments.

I'm closing this pull request. Thanks for taking the time to look at it.

Reply:
Note that I feel that adding at least the root cause message would be an improvement in any case, it's definitely not great to not have any kind of clue of what was wrong.

Reply:
I'm reopening the pull request this time logging only the root cause.

Also I took the time to read https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/ and properly configure my IDE.